src/Resolver.php

```diff
@@ -124,6 +126,27 @@ public function decodePage(ResponseInterface $response)
         return false;
     }
 
+    public function resolveMenus()
+    {
+        $response = $this->api->getMenus();
+        if (empty($response)) return false;
+
+        $body = (string) $response->getBody();
+        $results = [];
+        if ($response->getHeaderLine('Content-Type') == "application/json") {
+            $menus = json_decode($body, true);
+            foreach ($menus as $menu) {
+                $r = $this->api->getMenuById($menu['id']);
+                $b = (string) $r->getBody();
+                if ($r->getHeaderLine('Content-Type') == "application/json") {
+                    $m = json_decode($b, true);
+                    array_push($results, $m);
+                }
+            }
+        }
+        return $results;
+    }
```

Comment:
@merricks this I could use feedback on in particular. I feel like it makes more sense to update the api to provide this data in a single call, but not sure what the considerations around that are.

Reply:
Is the issue here that the menu list endpoint doesn't include the links in it? Even if we don't change the default response, it would be a good option to be able to pass in a flag that says like "and give me the links too"—it's a pattern we've used before to give some control over what is being asked for from the API. See for example line 69 in `PublicFeedController.php`.